core/src/main/scala/org/http4s/parser/AdditionalRules.scala

```diff
@@ -221,4 +221,10 @@ private[http4s] object AdditionalRules {
       case _: NumberFormatException => None
     }
   }
+
+  val ServerAgentParser: P[(ProductId, List[ProductIdOrComment])] = {
```

Comment:
Can we get this on the companion of `ProductIdOrComment`?  Putting models in this package gets us circular dependencies.

Reply:
https://github.com/http4s/http4s/pull/4208/commits/c6a368b9b9a64f5744880ec3628ed806e93e3f5a